tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/ContactEntity.cs

```diff
@@ -23,5 +25,17 @@ public sealed class ContactEntity : EntityBase
                 @Phone
             );
             SELECT SCOPE_IDENTITY();";
+
+        public static async Task<ContactEntity> FetchContactByContactId(string connectionString, int? contactId)

```

Comment:
Same as the addressId comment above. Does this need to execute the database query if `contactId` is null?